packaging/fedora/snapd.spec

```diff
@@ -508,10 +508,15 @@ sed -e "s/-Bstatic -lseccomp/-Bstatic/g" -i cmd/snap-seccomp/*.go
 %gobuild -o bin/snap-seccomp $GOFLAGS %{import_path}/cmd/snap-seccomp
 
 %if 0%{?with_selinux}
-# Build SELinux module
-pushd ./data/selinux
-make SHARE="%{_datadir}" TARGETS="snappy"
-popd
+(
+%if 0%{?rhel} == 7
+    M4PARAM='-D distro_rhel7'
+%endif
+    # Build SELinux module
+    cd ./data/selinux
+    # pass M4PARAM in env so that make can still manipulate it freely
```

Comment:
Perhaps referring to the make override feature would be more explanatory:
https://www.gnu.org/software/make/manual/html_node/Override-Directive.html

Reply:
Yes, this would be helpful. 👍